spec/features/practice_editor/adoptions_spec.rb

```diff
@@ -330,32 +316,6 @@ def create_adoption(status, facility_index)
       before do
         create_adoption('completed', 0)
       end
-
-      context 'on page load' do
-        it 'should display a tooltip with adoption status definitions if the user hovers over the tooltip icon within the new adoption form' do
-          new_adoption_form_tooltip_flow
-        end
-
-        it 'should display a tooltip with adoption status definitions if the user hovers over the tooltip icon within an existing adoption form' do
-          open_successful_adoption_accordion
-          click_button('ME: Caribou VA Clinic')
-          tooltip_expectations
-        end
-      end
-
-      context 'after ajax call' do
-        it 'should display a tooltip with adoption status definitions if the user hovers over the tooltip icon within the new adoption form' do
-          create_adoption('completed', 1)
-          new_adoption_form_tooltip_flow
-        end
-
-        it 'should display a tooltip with adoption status definitions if the user hovers over the tooltip icon within an existing adoption form' do
-          create_adoption('completed', 2)
-          open_successful_adoption_accordion
-          click_button('NY: Yonkers VA Clinic')
-          tooltip_expectations
-        end
-      end
```

Comment:
Can we refactor these tests instead of removing them?

Reply:
We can..  :)

Reply:
Did we ever refactor these, or add new ones that test the new modals?

Reply:
We added a new one for that Status modal in the adoption form but we were able to refactor the other ones where the tooltip icon was showing but now displays a modal.